test/test262.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+cd "$(dirname "$0")"
+if [ ! -d "tc39" ]; then
+  git clone --depth=1 https://github.com/tc39/proposal-temporal.git tc39
+else
+  cd ./tc39
+  git fetch origin
+  git merge --ff-only origin/main
+fi
+
+cd ../ && TEST262=1 npm run build
+cd tc39/polyfill && npm install
```

Comment:
The only thing I might change about this is having to `npm install` the tc39 repo, maybe we should put whatever dependencies are needed into this repo's `package.json`? But I don't know if that's possible.

Reply:
We could probably list it as a devDependency.

That being said, [the package doesn't include the test262 files anyways](https://github.com/tc39/proposal-temporal/blob/33ef37bb30b2c5081b2fe8fa119e39b4bdae0c45/polyfill/package.json#L61-L65), and it would also mean having to wait for that package to release to pickup new tests. Hopefully these tests will move into the test262 repo and presumably they release those relatively frequently?

Reply:
I don't think they release them at all - the latest state is always on the main branch and engines pull that in at whatever cadence suits them.

Reply:
I don't think having those dependencies listed in our package.json is going to be useful, given that the script in the tc39 repo won't look those dependencies there. That being said, I do hope that this is a relatively temporary solution and that we instead check in our own version of this (or, have a third repo with this config setup in it we can add as a devDependency for this / other projects that want to run the Temporal tests?), instead of cloning the repo.

That being said, I'd much rather have test262 coverage on this repo working and sort this out later.